### ISSUE TITLE
5% Higher Than What?!??!

### ISSUE THREAD

**Issue by DeathRowKitty** [Grimy/Grimy.github.io#14]
http://puu.sh/vHAhi/e8e46e0132.png  I thought this was a pretty funny "bug".  Maybe add a special case to avoid this? :p

**Comment by Grimy** [comment]
Not really a bug. You can make multiple z238 maps with different loot values. I agree that it sounds weird, though.

**Comment by DeathRowKitty** [comment]
I get that you can have multiple z238 maps, but if you were to make a z238 map with better loot, running it in D formation wouldn't suddenly be better than running it in S.  I put "bug" in quotation marks because it's presumably not wrong that running a higher loot z238 map in D would be better than running a lower loot z238 map in S, but it's kinda silly for it to say that when there's no situation (in this instance) in which being in D formation in a z238 map would give higher loot than being in S.

Either way, not a particularly important thing and I got a nice laugh out of it giving me that advice

**Comment by jrynd** [comment]
It is a wording issue. The best map-formation combination gives this much, the second-best map-formation combination is only 5% behind.

**Comment by jrynd** [comment]
So here's my proposal: It still tells you the ideal map level to farm at. But its secondary suggestion should be "If you don't have enough fragments to make a max loot map at this level, the best you can do is x% or more below max loot, but you do have enough for a max loot map at this lower level, do it." Or something similar anyway. It should never suggest a map at the same or higher level as an alternate, because there's no way to increase the loot.

**Comment by Grimy** [comment]
> If you don't have enough fragments to make a max loot map at this level, the best you can do is x% or more below max loot, but you do have enough for a max loot map at this lower level, do it.

That’s way too wordy, and also inaccurate (lack of fragments isn’t the only reason why you wouldn’t want to make a max loot map). I like the current formulation better. Sure, it sounds silly when the top two options are the same zone, but that’s an extremely rare case (the optimal S zone is usually 3 below the optimal D zone).

> It should never suggest a map at the same or higher level as an alternate

I get the reasonning behind “at the same level”, but what about higher levels? It’s completely possible to have a higher level map with higher loot; this happens to me almost every run.
